scripts/nb-tester/requirements.txt

```
@@ -2,7 +2,10 @@
 # stable build.
 
 qiskit[all]~=2.1.2
-qiskit-ibm-runtime~=0.40.1
+# adding this line to test Roy's code and get output.  It should be removed before merging.
+qiskit-ibm-runtime @ git+https://github.com/RoyElkabetz/qiskit-ibm-runtime.git@re-sched-vis
+#qiskit-ibm-runtime~=0.43.0
```

```suggestion
qiskit-ibm-runtime~=0.43.0
```

0.43.0 was just released 🎉

---

Done in https://github.com/Qiskit/documentation/pull/3950/commits/70d90a6356188f54a1e8eac686e60f740f6ecd6a 🎉